dashboard/app/mailers/follower_mailer.rb

```diff
@@ -1,4 +1,5 @@
-class FollowerMailer < ActionMailer::Base
+class FollowerMailer < ApplicationMailer
+  include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`

Reply:
Oh that's embarrassing, thanks for catching that.